autogpt/processing/text.py

```diff
@@ -1,6 +1,5 @@
 """Text processing functions"""
-from typing import Dict, Generator, Optional
-
```

Comment:
Unnecessary refactoring, please undo

Reply:
I undid the reorder but still need the `List` for the `split_long_paragraph` function